vuu-ui/package.json

```diff
@@ -68,7 +68,7 @@
     "eslint-config-prettier": "8.7.0",
     "eslint-plugin-cypress": "^2.13.3",
     "eslint-plugin-prettier": "4.2.1",
-    "eslint-plugin-react": "7.32.2",
```

Comment:
why this change ?

Reply:
Think it was an auto dependency update, but I'll switch it back manually next push!